app/models/sales_log.rb

```diff
@@ -396,6 +396,10 @@ def is_bedsit?
     proptype == 2
   end
 
+  def is_beds_inferred?
+    form.start_year_2025_or_later? && is_bedsit?
+  end
```

Comment:
Ah, just noticed - I'm expected some code that actively sets beds to 1 if this is true (probably in derived variables, I think that's where it is for lettings?)

Reply:
Ah yes, that's the missing link, good spot. I'll add that in.